### DISCUSSION TITLE
babelRegister.setOnlyMap

### DISCUSSION THREAD

**Discussion by wangsanshao** [umijs/umi#12145] [Q&A]
umi3 的babelRegister.setOnlyMap 升级umi4 该怎么处理

**Comment by fz6m** [comment]
umi 4 没有 `setOnlyMap` ，默认 babel 转换除了 `node_modules` 以外的所有 js 、ts 文件，如果需要单独转换某些第三方包，就把包名配到 [`extraBabelIncludes`](https://umijs.org/docs/api/config#extrababelincludes) 里就可以了（一般用于需要 es5 兼容的场景，转换那些只发布了 es6 的包）。
